sktime/transformations/panel/tsfresh.py

```diff
@@ -616,9 +616,6 @@ def fit_transform(self, X, y=None):
                 then the return is a `Panel` object of type `pd-multiindex`
                 Example: i-th instance of the output is the i-th window running over `X`
         """
-        self.reset()
-        if y is None:
-            raise ValueError("SupervisedIntervals requires `y` in `fit`.")
         X, y, metadata = self._check_X_y(X=X, y=y, return_metadata=True)
```

Comment:
In `fit()` this reads:

`X_inner, y_inner = self._check_X_y(X=X, y=y)`

(without the metadata). Maybe unify both `fit`and `fit_transform`?

Reply:
While `fit` does not need the metadata, `transform` does for 
```
        # convert to output mtype
        if not hasattr(self, "_output_convert") or self._output_convert == "auto":
            X_out = self._convert_output(Xt, metadata=metadata)
        else:
            X_out = Xt
```